pkgs/servers/mattermost/default.nix

```diff
@@ -76,6 +76,9 @@ let
         mattermost/fonts \
         mattermost/templates \
         mattermost/config
+
+      # For some reason a bunch of these files are +x...
+      find $out -type f -exec chmod -x {} \;
```

Comment:
Hopefully we won't need to do this when we're building the frontend on our own. Eventually...